pkgs/development/libraries/imgui/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, lib, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
```

Comment:
I think `rec` in here now might not be necessary is that correct?

Otherwise I should clarify that avoiding recursion for making repo refer to pname is not against code style as far as I know. It would be just my personal preference.

Reply:
> I think `rec` in here now might not be necessary is that correct?

If I remove the rec, it complains about version being undefined

> Otherwise I should clarify that avoiding recursion for making repo refer to pname is not against code style as far as I know. It would be just my personal preference.

I completely understand it, I just feel it is safer to use the direct name in case there are any changes on the repo name (because it is also called Dear Imgui).

Reply:
ah right! sorry I missed that. I think for version this makes more seance though so I would keep it.